spec/support/good_job_test_helper.rb

```diff
@@ -0,0 +1,33 @@
+require 'fileutils'
+
+module GoodJobTestHelper
```

Comment:
This class name is a little generic. It's specifically for setting up and tearing down a full rails app. Unfortunately there is already a `test_app`, so maybe `ExampleAppHelper` ("example" in this case meaning a RSpec example).